src/FormlyMapper/FormlyField/BirthdayField.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class BirthdayField extends FormlyField
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFieldType()
+    {
+        return 'birthday';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function buildFieldTypeConfiguration(): void
+    {
+        $this->formlyFieldConfiguration['templateOptions']['type'] = $this->getFieldType();
+
+        $order = 'desc';
+        if(isset($this->fieldConfiguration['options']['order'])) {
+            $tmpOrder = strtolower($this->fieldConfiguration['options']['order']);
```

Comment:
```suggestion
            $fieldConfigurationOrder = strtolower($this->fieldConfiguration['options']['order']);
```

Try to avoid using `tmp`. as a variable name, you should always try to use expressive names.